pubspec.yaml

```diff
@@ -55,6 +55,7 @@ dependencies:
   emoji_picker_flutter: ^4.0.0-dev.2
   reorderable_grid: ^1.0.10
   local_auth: ^2.3.0
+  sqflite_common_ffi: ^2.3.4+4
```

Comment:
Accordingly to the readme of the package, this goes to dev_dependencies. For making it work on android and iOS another library seem needed https://pub.dev/packages/sqlite3_flutter_libs 

So I am not sure if it is needed to keep both `sqflite_common_ffi` and `sqflite`.